elements/lisk-chain/src/index.ts

```diff
@@ -42,5 +42,6 @@ export type {
 } from './types';
 export { Slots } from './slots';
 export { readGenesisBlockJSON, getValidators, getAccountSchemaWithDefault } from './utils';
+export * as testing from './testing';
```

Comment:
I am not sure if that also applied for a node module, but what if we put this mock into `__mocks__` directory as per Jest specification for manual mocks. We can still export if we want.

Reply:
Im not sure if it works with `__mocks__` within `node_module`, but we can add it later if it works?
But anyway, i think we should expose this as testing